app/src/main/java/com/pysiak/simple/clock/ui/ClockPresenter.java

```diff
@@ -0,0 +1,35 @@
+package com.pysiak.simple.clock.ui;
+
+import android.content.Context;
+import android.support.v4.app.Fragment;
+import com.pysiak.simple.clock.R;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ClockPresenter {
```

Comment:
Add javadoc to class

Reply:
Please review my PR fix code style, add javadoc: